app/tests/unit/components/snackbar.spec.js

```diff
@@ -27,7 +29,8 @@ describe('@/components/Snackbar.vue', () => {
     expect(wrapper.vm.show).toBe(false)
   })
 
-  it('contains a refresh button by default', async () => {
+  // TODO: @Anya determine if we still need this button. Skipping for now
+  it.skip('contains a refresh button by default', async () => {
```

Comment:
I agree we can leave off the refresh button. I'm still not sure about the close one, in case we ever run into issues with the infinite snackbar not going away, but I'm fine with taking it off for now

Reply:
> I agree we can leave off the refresh button. I'm still not sure about the close one, in case we ever run into issues with the infinite snackbar not going away, but I'm fine with taking it off for now

sure, I'll leave the comment as is just in case we need to and I'll get rid of the refresh